### ISSUE TITLE
forward prefetch error: ransformers version 

### ISSUE THREAD

**Issue by sallywang147** [tatsu-lab/stanford_alpaca#115]
Hi,

thanks for your great work! which transformers did you use for training? I tried to reproduce the result with transformers 4.28.0, but I got the following error:
 ` __init__() got an unexpected keyword argument 'forward_prefetch'` in transformers/trainer.py. Thank you for your help.

**Comment by 447428054** [comment]
same error

**Comment by Jacksonlark** [comment]
same error

TypeError: __init__() got an unexpected keyword argument 'forward_prefetch'

@sallywang147 @447428054 Are you fix the error?

**Comment by Jacksonlark** [comment]
May be torch version is not compatible with transformers

https://pytorch.org/docs/1.12/fsdp.html

pytorch version <= 1.12, The FullyShardedDataParallel not have forward_prefetch

**Comment by 447428054** [comment]
@Jacksonlark @sallywang147 i fix it by installing transformers==4.26.1, I'll post the code in a moment

**Comment by Jacksonlark** [comment]
> 4.26.1

@447428054 Using transformers==4.26.1, How do you load llama model?

**Comment by Jacksonlark** [comment]
> @Jacksonlark @sallywang147 i fix it by installing transformers==4.26.1, I'll post the code in a moment

@447428054 Using transformers==4.26.1, How do you load llama model?

**Comment by sallywang147** [comment]
@447428054 yeah same question as @Jacksonlark. I can't load the model without the latest transformer

**Comment by Jacksonlark** [comment]
> @447428054 yeah same question as @Jacksonlark. I can't load the model without the latest transformer

You can check new branch from transformers==4.26.1 and migration llama code, then pip this branch.

**Comment by sallywang147** [comment]
Ah thanks! Actually witch torch version worked for me. I still used the latest transformer 4.28.0dev0.

**Comment by srikanthsrnvs** [comment]
@sallywang147 How are you getting it to fine-tune? My GPU's seem to freeze up after a few minutes, I see no communication or usage, just memory usage and my entire pipeline seems stuck

**Comment by 447428054** [comment]
> > @447428054 yeah same question as @Jacksonlark. I can't load the model without the latest transformer
> 
> You can check new branch from transformers==4.26.1 and migration llama code, then pip this branch.

Yeah, The cod is as follows:https://github.com/447428054/stanford_alpaca_llama.git

**Comment by Camellia1110** [comment]
> 
@sallywang147 Is your torch version 1.3.1

**Comment by Camellia1110** [comment]
@sallywang147Is your torch version 1.13.1？

**Comment by Hiusam** [comment]
Same question. What version of torch?
